### DISCUSSION TITLE
How to deploy `.env` file?

### DISCUSSION THREAD

**Discussion by bilogic** [deployphp/deployer#3803] [Help needed]
1. I have dev, staging and production environments and all of them reference a `.env` file but have different contents
2. Currently, I use deployer to update production using the files from staging
3. How do I also deploy `.env` without having to manually login to production and modify it? Is there a built-in way?

**Comment by ogrosko** [comment]
Create task and run following:

`run('cp {{release_path}}/.env.' . get('labels')['stage'] . ' {{release_path}}/.env');`
